tests/unit_test/test_curve.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import unittest
+from beprof.curve import Curve
+
+
+class TestCurve(unittest.TestCase):
+    def setUp(self):
+        self.pre = Curve([[0.0, 0], [5, 5], [10, 0]])
```

Comment:
Curve is meant to be more general than a profile - it doesn't have to first rise then fall.